mrp_request/models/mrp_request.py

```diff
@@ -6,7 +6,7 @@
 
 
 class MrpProductionRequest(models.Model):
-    _name = "mrp.production.request"
+    _name = "mrp.request"
```

Comment:
The rename of the module will be done in the OpenUpgrade scripts. But this model rename has to be handled in this module adding custom migrations scripts.

Reply:
@alan196 if you don't know hot do that, then let's merge the PR and then later I will do one to add the migration scripts.